frontend/src/model/beta/images/UploadNewImageDialog.tsx

```diff
@@ -0,0 +1,127 @@
+      method: 'POST',
+    }).then((res) => res.json())
+
+    return token
+  }
+
+  const showToken = async () => {
+    const token = await regenerateToken()
+    setDisplayedToken(token)
+    setDisplayToken(true)
+  }
+
+  if (isUiConfigError) {
+    return <MessageAlert message={isUiConfigError.info.message} severity='error' />
+  }
+
+  if (isCurrentUserError) {
+    return <MessageAlert message={isCurrentUserError.info.message} severity='error' />
+  }
+
+  return (
+    <>
+      {(isUiConfigLoading || isCurrentUserLoading) && <Loading />}
+      {uiConfig && currentUser && (
+        <Dialog open={open} onClose={handleClose}>
+          <DialogTitle color='primary'>Uploading an Image for this Model</DialogTitle>
+          <DialogContent>
+            <Stack spacing={2}>
+              <Typography fontWeight='bold'>User authentication token</Typography>
+              <Typography>Use the token below to authenticate when you try and run the docker login command</Typography>
+              <Stack direction='row'>
+                <Button sx={{ mr: 2 }} variant='outlined' onClick={showToken} data-test='showTokenButton'>
+                  Regenerate Token
+                </Button>
+                <Box
+                  sx={{
+                    backgroundColor: theme.palette.container.main,
+                    pr: 2,
+                    pl: 2,
+                    display: 'flex',
+                    mr: 1,
+                  }}
+                >
+                  <Box component={Stack} direction='column' justifyContent='center'>
+                    <Typography variant='body1' data-test='dockerPassword'>
+                      {displayToken ? displayedToken : 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxxx'}
+                    </Typography>
+                  </Box>
+                </Box>
+                <Tooltip title='Regenerate & Copy to clipboard'>
+                  <IconButton
+                    onClick={async () => {
+                      const token = await regenerateToken()
+                      setDisplayedToken(token)
+                      navigator.clipboard.writeText(token)
+                    }}
+                    aria-label='regenerate and copy to clipboard'
+                  >
+                    <ContentCopy />
+                  </IconButton>
+                </Tooltip>
+              </Stack>
+              <Stack spacing={1}>
+                <Typography fontWeight='bold'>Logging in</Typography>
+                <Stack spacing={2}>
+                  <CodeLine line={`docker login ${uiConfig.registry.host} -u ${currentUser.id}`} />
+                </Stack>
+              </Stack>
+              <Stack spacing={1}>
+                <Typography fontWeight='bold'>Pushing an image to the registry</Typography>
+                <Stack spacing={2}>
+                  <CodeLine line={`docker tag <image> ${window.location.host}/<namespace>/<model>:<version>`} />
```

Comment:
Since the `<namespace>` is the model ID, can we populate that?

Reply:
Can we also rename `<model>` to `<name>`? It's how that part of the Docker URI is referred to in the Docker docs and is less confusing as the namespace is the Model ID